plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityCharged.java

```diff
@@ -35,7 +38,7 @@ private EntityCharged(EntityTag entity) {
 
     @Override
     public String getPropertyString() {
-        return ((WitherSkull) entity.getBukkitEntity()).isCharged() ? "true" : "false";
+        return isCharged() ? "true" : null;

```

Comment:
That `false` to `null` is a questionable change. General rule of thumb with something like this - there's probably a reason it was the way it was, and unless you either
A: know exactly what that reason was, and know it doesn't apply any longer or was wrong, or
B: have separately and confidently confirmed that the way you're changing it to is more correct,
then don't change it.